packages/espresso/README.md

```diff
@@ -0,0 +1,121 @@
+# espresso
+
+Provides bindings for Espresso tests of Flutter Android apps.
+
+## Installation
+
+Add ```android:usesCleartextTraffic="true"``` in the ```<application>``` in the AndroidManifest.xml
+of the Android app used for testing.
+
+Add dependencies to your build.gradle:
+
+```
```

Comment:
I think github markdown supports syntax highlighting https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks